test/functional/rpc_packages.py

```diff
@@ -325,18 +324,11 @@ def test_submit_child_with_parents(self, num_parents, partial_submit):
         self.generate(node, 1)
 
     def test_submitpackage(self):
-        node = self.nodes[0]
-
         self.log.info("Submitpackage valid packages with 1 child and some number of parents")
         for num_parents in [1, 2, 24]:
             self.test_submit_child_with_parents(num_parents, False)
             self.test_submit_child_with_parents(num_parents, True)
 
-        self.log.info("Submitpackage only allows packages of 1 child with its parents")
```

Comment:
should just leave this example in in the positive sense of it being accepted?

Reply:
Re-added